docs/units/type_hints.rst

```diff
@@ -72,8 +65,6 @@ Multiple Annotations
 Multiple Quantity and unit-aware |Quantity| annotations are supported using
 :class:`~typing.Union` or :class:`~typing.Optional`
 
-.. doctest-skip::
-
    >>> T.Union[Quantity[u.m], None]
```

Comment:
Do you want to change this one to `Quantity[u.m] | None` too?

Reply:
✅